packages/calling/src/Metrics/index.test.ts

```diff
@@ -534,4 +540,37 @@ describe('CALLING: Metric tests', () => {
       );
     });
   });
+
+  describe('Calling_Sdk_Version fallback test', () => {
```

Comment:
Added test for fallback. Currently just testing with one particular metric.

Reply:
Yes, just one is fine. Make the env object return null or empty object, that's when actual fallback will happen